src/golang/lib/models/shared/integration_config.go

```diff
@@ -53,6 +53,12 @@ type EmailConfig struct {
 	Level    NotificationLevel `json:"level" yaml:"level"`
 }
 
+type SlackConfig struct {
+	Token    string            `json:"token" yaml:"token"`
+	Channels []string          `json:"channels" yaml:"channels"`
+	Level    NotificationLevel `json:"level" yaml:"level"`
```

Comment:
This config corresponds to a single integration right? If I want to set up different channels with different notification levels, will that require me to set up multiple slack integrations on the UI?

Reply:
If so, would it be better to have a notification level per channel, so you'd only have to set it up once.

Reply:
Yeah this makes sense! But :) - we actually discussed this in a few deliverable meetings, and we explicitly decided to just have one level for all slack channels to start with.

It would require some data migration if we want to switch gear, but there shouldn't be a lot of burden comparing to the additional UI work. So I don't see a strong motivation to just enable this in backend as well.